### PR TITLE
Collective Overview (preview feature)

### PR DESCRIPTION
**Pull request by gustavlrsn**
For https://github.com/opencollective/opencollective/issues/7180

# Description

First iteration of a Collective Overview in Dashboard, hidden behind a closed beta preview feature.

Still missing activities.

# Screenshots

image

Toggle in Preview features modal, available internally
image

### CODE REVIEW

**Comment by gustavlrsn** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #9907** <a href="https://app.graphite.dev/github/pr/opencollective/opencollective-frontend/9907?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/opencollective/opencollective-frontend/9907?utm_source=stack-comment).

File: components/dashboard/DashboardSection.tsx [hunk 1]
```diff
@@ -84,6 +86,9 @@ const DashboardSection = ({ account, isLoading, section, subpath }) => {
     if (section === SECTIONS.TRANSACTIONS && useNewTransactionsPage) {
       DashboardComponent = AccountTransactions;
     }
+    if (section === SECTIONS.OVERVIEW && isIndividualAccount(account)) {
```
Comment (kewitz) [inline components/dashboard/DashboardSection.tsx:89]: If SECTIONS is no longer a 1:1 map, we should consider turning SECTIONS into a function that resolves the correct component.

File: components/dashboard/sections/Overview/TodoList.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Receipt } from 'lucide-react';
+import { FormattedMessage } from 'react-intl';
+
+import { getDashboardRoute } from '../../../../lib/url-helpers';
+
+import Link from '../../../Link';
+
+export const TodoList = ({ account }) => {
```
Comment (kewitz) [inline components/dashboard/sections/Overview/TodoList.tsx:9]: We should type this props and require pending expenses.

**Review by kewitz — APPROVED** [review]
Awesome work!

File: components/dashboard/DashboardSection.tsx (hunk 1 above)
Comment (gustavlrsn) [reply components/dashboard/DashboardSection.tsx:89]: Yeah, that'd make sense! Will save this for a follow up

**Comment by gustavlrsn** [comment]
@kewitz thanks. Pushed an additional fix to not include children for child accounts (to fix the balances bug), and set the Received and Spent stats to use `net: true` to get the net amount which seems more appropriate (and matching with the balance)

**Comment by gustavlrsn** [comment]
### Merge activity

* **Jan 24, 12:00 PM**: @gustavlrsn merged this pull request with [Graphite](https://app.graphite.dev/github/pr/opencollective/opencollective-frontend/9907).

**Pull request merged**